content/docs/command-reference/remote/add.md

```diff
@@ -198,6 +198,30 @@ $ dvc remote add -d myremote 'azure://'
 
 <details>
 
+### Click for Dropbox
+
+To start using a Dropbox remote, first add it with a
+[valid URL format](/doc/user-guide/setup-dropbox-remote#url-format). Then use
+any DVC command that needs to connect to it (e.g. `dvc pull` or `dvc push` once
+there's tracked data to synchronize). For example:
+
+```dvc
+$ dvc remote add -d myremote dropbox://my-project
+$ dvc push  # Assuming there's data to push
+
+Go to the following link in your browser:
+
+    https://www.dropbox.com/oauth2/authorize?response_type=code&client_id=<client-id>&token_access_type=offline # ... copy this link
+
+Enter verification code: # <- enter resulting code
+```
+
+> Please note [Privacy Policy (Dropbox API)](/doc/user-guide/dropbox-privacy).
```

Comment:
One or both of the paragraphs found in the GDrive section of this same doc should probably be here too? With the appropriate changes, of course:

![image](https://user-images.githubusercontent.com/1477535/104078037-b7519280-51e1-11eb-988b-fdc20230c63c.png)

See https://dvc.org/doc/command-reference/remote/add#supported-storage-types

Reply:
BTW:
````suggestion
Enter verification code: # <- enter resulting code
```

> Please note our [Privacy Policy (Dropbox API)](/doc/user-guide/dropbox-privacy).
````

Reply:
The first one (the one you highlighted) should be included. The second one probably not, as Dropbox API exposes file checksum (GDrive doesn't).